client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -456,133 +346,172 @@ function TitanMailMeta( { item, isRenewal } ) {
-	hasDomainsInCart,
-	createUserAndSiteBeforeTransaction,
-	isPwpoUser
-) {
-	const modalCopy = {};
+	product: string,
+	translate: ReturnType< typeof useTranslate >,
+	hasDomainsInCart: boolean,
+	createUserAndSiteBeforeTransaction: boolean,
+	isPwpoUser: boolean
+): ModalCopy {
 	const productType = product === 'plan' && hasDomainsInCart ? 'plan with dependencies' : product;
 
 	switch ( productType ) {
-		case 'plan with dependencies':
-			modalCopy.title = translate( 'You are about to remove your plan from the cart' );
+		case 'plan with dependencies': {
+			const title = String( translate( 'You are about to remove your plan from the cart' ) );
+			let description = '';
 
 			if ( createUserAndSiteBeforeTransaction ) {
-				modalCopy.description =
-					'When you press Continue, we will remove your plan from the cart. Your site will be created on the free plan when you complete payment for the other product(s) in your cart.';
+				description = String(
+					translate(
+						'When you press Continue, we will remove your plan from the cart. Your site will be created on the free plan when you complete payment for the other product(s) in your cart.'
+					)
+				);
 			} else {
-				modalCopy.description = isPwpoUser
-					? translate(
-							'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan.'
-					  )
-					: translate(
-							'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. Since your other product(s) depend on your plan to be purchased, they will also be removed from the cart and we will take you back to your site.'
-					  );
+				description = String(
+					isPwpoUser
+						? translate(
+								'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan.'
+						  )
+						: translate(
+								'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. Since your other product(s) depend on your plan to be purchased, they will also be removed from the cart and we will take you back to your site.'
+						  )
+				);
 			}
-			break;
+			return { title, description };
+		}
 		case 'plan':
-			modalCopy.title = translate( 'You are about to remove your plan from the cart' );
-			modalCopy.description = createUserAndSiteBeforeTransaction
-				? 'When you press Continue, we will remove your plan from the cart.'
-				: translate(
-						'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan. We will then take you back to your site.'
-				  );
-			break;
+			return {
+				title: String( translate( 'You are about to remove your plan from the cart' ) ),
+				description: String(
+					createUserAndSiteBeforeTransaction
+						? translate( 'When you press Continue, we will remove your plan from the cart.' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'When you press Continue, we will remove your plan from the cart and your site will continue to run with its current plan.' )` *ES Score: 6*

Reply:
I don't know why these strings were not translated in the first place when they were added, but this is different because the site does not exist yet.